root/search/components/ReleaseResults.js

```diff
@@ -37,8 +37,16 @@ function buildResult(result, index) {
       <td>
         <ArtistCreditLink artistCredit={release.artistCredit} />
       </td>
-      <td>{release.combined_format_name}</td>
-      <td>{release.combined_track_count}</td>
+      <td>
+        {release.combined_format_name
+          ? release.combined_format_name
+          : '-'}
+      </td>
+      <td>
+        {release.combined_track_count
+          ? release.combined_track_count
+          : '-'}
```

Comment:
(Nitpick: `{release.combined_track_count || '-'}` is shorter and does the same thing.)